recipes/orange3-fairness/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "Orange3-Fairness" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 18f85525bf1a4b1cf6286a92051b6053ad745826311c037cd40482411b95824b
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.9,<3.12
+    - setuptools
+  run:
+    - aif360 >=0.5.0
+    - numpy <=1.23
```

I think it should be changed to 
```suggestion
    - numpy <1.24
```
to also include versions between 1.23.1 and 1.23.5.